.circleci/config.yml

```diff
@@ -144,7 +144,7 @@ jobs:
                     -kubecontext="kind-dc1" \
                     -secondary-kubecontext="kind-dc2" \
                     -debug-directory="$TEST_RESULTS/debug" \
-                    -consul-k8s-image=docker.mirror.hashicorp.services/hashicorpdev/consul-k8s:latest
+                    -consul-k8s-image=gcr.io/nitya-293720/consul-k8s-dev:metrics30
```

Comment:
On the feature metrics branch, should we use this image so we can see acceptance tests passing once we merge this and #834 in? (I was also thinking about this from connect service metrics pr ( #834 )).

Reply:
That makes sense 👍

Reply:
Let's merge this on #834 into the feature branch and then create a potential PR that has the latest images.